doc/development_workflow.md

```diff
@@ -0,0 +1,60 @@
+# Development process in Status Open Bounty
+
+We have a continuously deployed version tracking the `develop` branch live at [test environment](https://openbounty.status.im:444). It uses the [Ropsten](https://ropsten.io/) Ethereum testnet. Any one is welcome to use it and contribute to Open Bounty.
+ 
+Any help is greatly appreciated!
+Currently we use two projects -  `Pipeline For Issues` and `Pipeline For Pull Requests`, issues and pull requests in our repository are passing through all or several stages described below.
+
+## Pipeline For Issues
+
+Issues are added to this project by product owner.
+
+#### To define
+This is backlog for all features/issues/enhancements which we want to include to development process. 
+
+All issues here should be marked with:
+* type - `bug`, `tech-debt`,` enhancement`, `proposal` labels
+* priority - `Prio: high`, `Prio: med`, `Prio: low` labels
+
+#### In Bounty
```

Comment:
I think that it would be better to position it at the right of `To Develop`, since a bounty issue is already defined, designed, and ready for development, but we just decide to put it as a bounty at that point.

Reply:
Hm, may be you are right. I just wondering - we have a lot of redundant issues in `To develop`  column which are intended mostly for external contributors. If all developers are happy to have all these issues in `To develop` - I don't see any problem to remove the column

Reply:
Oh, I didn't mean to remove the column `In Bounty`, but just to move it at the right of `To Develop`.

Reply:
Ok, agree